### ISSUE TITLE
Starting a video through UPnP / DLNA remote control is only working once after reboot

### ISSUE THREAD

**Issue by arminfuerst** [xbmc/xbmc#25417]
### Describe the bug
Here is a clear and concise description of what the problem is:

I recently had to upgrade from LibreELEC 9 (=KODI 19) to LibreELEC 12 (=KODI 21). I am mainly using these devices to view videos from Synology Video Station which remote controls KODI through UPnP / DLNA.
Since the upgrade to KODI 21, I can only start one video after rebooting the device, any additional try to start a video results in the error "One or more items failed to play.Check the log for more information about this message."
With LibreELEC 9 I never experienced this problem.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

I expect to be able to play more than one video after a reboot of the device / start of KODI.

## Actual Behavior

Starting from the 2nd try, I always get the error "One or more items failed to play. Check the log for more information about this message."

## Possible Fix

Not a fix, but a workaround: reboot the device after having played a video to be able to play another video.

### To Reproduce

1. Configure KODI to "Enable UPnP support" and to "Allow remote control via UPnP"
![image](https://github.com/xbmc/xbmc/assets/29121472/a6ab6b07-7556-4e27-828c-d2694d0d3657)
2. Reboot the device
3. Start a video through UPnP / DLNA. It doesn't matter whether the video is stopped after a short time or the whole video is played.
4. Try to start another video through UPnP / DLNA.

I was also able to reproduce the behavior with KODI 20.5 running on Debian.

### Debuglog

The debuglog can be found here:

**Startup of device before first play**
```
2024-06-30 18:31:21.511 T:893      info <general>: -----------------------------------------------------------------------
2024-06-30 18:31:21.512 T:893      info <general>: Starting Kodi (21.0 (21.0.0) Git:21.0-Omega). Platform: Linux ARM 32-bit
2024-06-30 18:31:21.512 T:893      info <general>: Using Release Kodi x32
2024-06-30 18:31:21.512 T:893      info <general>: Kodi compiled 2024-04-25 by GCC 13.2.0 for Linux ARM 32-bit version 6.6.28 (394780)
2024-06-30 18:31:21.512 T:893      info <general>: Running on BCM2835 with LibreELEC (official): 12.0.0, kernel: Linux ARM 32-bit version 6.6.28
2024-06-30 18:31:21.512 T:893      info <general>: FFmpeg version/source: 6.0.1
2024-06-30 18:31:21.512 T:893      info <general>: Host CPU: ARMv7 Processor rev 4 (v7l), 4 cores available
2024-06-30 18:31:21.512 T:893      info <general>: ARM Features: Neon enabled
2024-06-30 18:31:21.512 T:893      info <general>: special://xbmc/ is mapped to: /usr/share/kodi/
2024-06-30 18:31:21.512 T:893      info <general>: special://xbmcbin/ is mapped to: /usr/lib/kodi
2024-06-30 18:31:21.512 T:893      info <general>: special://xbmcbinaddons/ is mapped to: /usr/lib/kodi/addons
2024-06-30 18:31:21.512 T:893      info <general>: special://masterprofile/ is mapped to: /storage/.kodi/userdata
2024-06-30 18:31:21.512 T:893      info <general>: special://envhome/ is mapped to: /storage
2024-06-30 18:31:21.512 T:893      info <general>: special://home/ is mapped to: /storage/.kodi
2024-06-30 18:31:21.512 T:893      info <general>: special://temp/ is mapped to: /storage/.kodi/temp
2024-06-30 18:31:21.512 T:893      info <general>: special://logpath/ is mapped to: /storage/.kodi/temp
2024-06-30 18:31:21.512 T:893      info <general>: Webserver extra whitelist paths: 
2024-06-30 18:31:21.512 T:893      info <general>: The executable running is: /usr/lib/kodi/kodi.bin
2024-06-30 18:31:21.512 T:893      info <general>: Local hostname: raspi-az
2024-06-30 18:31:21.512 T:893      info <general>: Log File is located: /storage/.kodi/temp/kodi.log
2024-06-30 18:31:21.512 T:893      info <general>: -----------------------------------------------------------------------
2024-06-30 18:31:21.512 T:893      info <general>: loading settings
2024-06-30 18:31:21.515 T:893      info <general>: special://profile/ is mapped to: special://masterprofile/
2024-06-30 18:31:21.552 T:893      info <general>: Loaded settings file from special://xbmc/system/advancedsettings.xml
2024-06-30 18:31:21.553 T:893      info <general>: Contents of special://xbmc/system/advancedsettings.xml are...
                                                   <?xml version="1.0" encoding="utf-8" ?>
                                                   <advancedsettings version="1.0">
                                                     <gputempcommand>/usr/bin/gputemp</gputempcommand>
                                                     <showexitbutton>false</showexitbutton>
                                                     <remotedelay>1</remotedelay>
                                                     <fanartres>720</fanartres>
                                                     <imageres>540</imageres>
                                                   </advancedsettings>
                                                   
2024-06-30 18:31:21.555 T:893      info <general>: No settings file to load (special://masterprofile/advancedsettings.xml)
2024-06-30 18:31:21.555 T:893      info <general>: Default Video Player: VideoPlayer
2024-06-30 18:31:21.555 T:893      info <general>: Default Audio Player: paplayer
2024-06-30 18:31:21.555 T:893      info <general>: Disabled debug logging due to GUI setting. Level 0.
2024-06-30 18:31:21.555 T:893      info <general>: Log level changed to "info"
2024-06-30 18:31:21.555 T:893      info <general>: CMediaSourceSettings: loading media sources from special://masterprofile/sources.xml
2024-06-30 18:31:21.560 T:893      info <general>: New Cache GUI Settings (replacement of cache in advancedsettings.xml) are:
                                                     Buffer Mode: 4
                                                     Memory Size: 20 MB
                                                     Read Factor: 4.00 x 
                                                     Chunk Size : 131072 bytes
2024-06-30 18:31:21.560 T:893      info <general>: creating subdirectories
2024-06-30 18:31:21.560 T:893      info <general>: userdata folder: special://masterprofile/
2024-06-30 18:31:21.560 T:893      info <general>: recording folder: 
2024-06-30 18:31:21.560 T:893      info <general>: screenshots folder: /storage/screenshots/
2024-06-30 18:31:21.651 T:893      info <general>: Running database version Addons33
2024-06-30 18:31:22.161 T:893      info <general>: CAddonMgr::FindAddons: audioencoder.kodi.builtin.aac v1.0.2 installed
2024-06-30 18:31:22.161 T:893      info <general>: CAddonMgr::FindAddons: audioencoder.kodi.builtin.wma v1.0.2 installed
2024-06-30 18:31:22.161 T:893      info <general>: CAddonMgr::FindAddons: game.controller.default v1.0.42 installed
2024-06-30 18:31:22.161 T:893      info <general>: CAddonMgr::FindAddons: game.controller.keyboard v1.1.40 installed
2024-06-30 18:31:22.161 T:893      info <general>: CAddonMgr::FindAddons: game.controller.mouse v1.0.26 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: game.controller.snes v1.0.40 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: inputstream.adaptive v21.4.10.1 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.audioengine v1.1.1 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.filesystem v1.1.8 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.general v1.0.5 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.gui v5.15.0 installed
2024-06-30 18:31:22.162 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.main v2.0.2 installed
2024-06-30 18:31:22.163 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.network v1.0.4 installed
2024-06-30 18:31:22.163 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.global.tools v1.0.4 installed
2024-06-30 18:31:22.163 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.audiodecoder v4.0.0 installed
2024-06-30 18:31:22.163 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.audioencoder v3.0.0 installed
2024-06-30 18:31:22.163 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.game v3.0.2 installed
2024-06-30 18:31:22.163 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.imagedecoder v3.0.1 installed
2024-06-30 18:31:22.164 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.inputstream v3.3.0 installed
2024-06-30 18:31:22.164 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.peripheral v3.0.2 installed
2024-06-30 18:31:22.164 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.pvr v8.3.0 installed
2024-06-30 18:31:22.164 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.screensaver v2.2.0 installed
2024-06-30 18:31:22.164 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.vfs v3.0.1 installed
2024-06-30 18:31:22.164 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.videocodec v2.1.0 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: kodi.binary.instance.visualization v4.0.0 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: kodi.resource v1.0.0 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: metadata.album.universal v3.1.9 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: metadata.artists.universal v4.3.11 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: metadata.common.allmusic.com v3.2.2 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: metadata.common.fanart.tv v3.6.4 installed
2024-06-30 18:31:22.165 T:893      info <general>: CAddonMgr::FindAddons: metadata.common.musicbrainz.org v2.2.4 installed
2024-06-30 18:31:22.166 T:893      info <general>: CAddonMgr::FindAddons: metadata.common.theaudiodb.com v2.0.3 installed
2024-06-30 18:31:22.166 T:893      info <general>: CAddonMgr::FindAddons: metadata.generic.albums v1.0.18 installed
2024-06-30 18:31:22.166 T:893      info <general>: CAddonMgr::FindAddons: metadata.generic.artists v1.0.18 installed
2024-06-30 18:31:22.166 T:893      info <general>: CAddonMgr::FindAddons: metadata.local v1.0.1 installed
2024-06-30 18:31:22.166 T:893      info <general>: CAddonMgr::FindAddons: metadata.themoviedb.org.python v3.0.1 installed
2024-06-30 18:31:22.166 T:893      info <general>: CAddonMgr::FindAddons: metadata.tvshows.themoviedb.org.python v1.6.6 installed
2024-06-30 18:31:22.167 T:893      info <general>: CAddonMgr::FindAddons: peripheral.joystick v21.1.11 installed
2024-06-30 18:31:22.167 T:893      info <general>: CAddonMgr::FindAddons: plugin.video.composite_for_plex v2.0.2 installed
2024-06-30 18:31:22.167 T:893      info <general>: CAddonMgr::FindAddons: plugin.video.orfon v1.0.0+beta installed
2024-06-30 18:31:22.167 T:893      info <general>: CAddonMgr::FindAddons: repository.libreelec.tv v12.0.0 installed
2024-06-30 18:31:22.167 T:893      info <general>: CAddonMgr::FindAddons: repository.xbmc.org v3.4.0 installed
2024-06-30 18:31:22.167 T:893      info <general>: CAddonMgr::FindAddons: resource.images.weathericons.default v1.1.9 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: resource.language.en_gb v2.0.2 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: resource.uisounds.kodi v1.0.1 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: screensaver.xbmc.builtin.black v1.0.34 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: screensaver.xbmc.builtin.dim v1.0.66 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: script.module.certifi v2023.5.7 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: script.module.chardet v5.1.0 installed
2024-06-30 18:31:22.168 T:893      info <general>: CAddonMgr::FindAddons: script.module.idna v3.4.0 installed
2024-06-30 18:31:22.169 T:893      info <general>: CAddonMgr::FindAddons: script.module.infotagger v0.0.8 installed
2024-06-30 18:31:22.169 T:893      info <general>: CAddonMgr::FindAddons: script.module.inputstreamhelper v0.6.1+matrix.1 installed
2024-06-30 18:31:22.169 T:893      info <general>: CAddonMgr::FindAddons: script.module.kodi-six v0.1.3.1 installed
2024-06-30 18:31:22.169 T:893      info <general>: CAddonMgr::FindAddons: script.module.pil v5.1.0 installed
2024-06-30 18:31:22.169 T:893      info <general>: CAddonMgr::FindAddons: script.module.pycryptodome v3.4.3 installed
2024-06-30 18:31:22.169 T:893      info <general>: CAddonMgr::FindAddons: script.module.pyxbmct v1.3.2+matrix.1 installed
2024-06-30 18:31:22.170 T:893      info <general>: CAddonMgr::FindAddons: script.module.requests v2.31.0 installed
2024-06-30 18:31:22.170 T:893      info <general>: CAddonMgr::FindAddons: script.module.routing v0.2.3+matrix.1 installed
2024-06-30 18:31:22.170 T:893      info <general>: CAddonMgr::FindAddons: script.module.six v1.16.0+matrix.1 installed
2024-06-30 18:31:22.170 T:893      info <general>: CAddonMgr::FindAddons: script.module.urllib3 v2.1.0 installed
2024-06-30 18:31:22.170 T:893      info <general>: CAddonMgr::FindAddons: script.plexmod v0.7.6 installed
2024-06-30 18:31:22.170 T:893      info <general>: CAddonMgr::FindAddons: service.libreelec.settings v12.0 installed
2024-06-30 18:31:22.171 T:893      info <general>: CAddonMgr::FindAddons: skin.estuary v4.0.0 installed
2024-06-30 18:31:22.171 T:893      info <general>: CAddonMgr::FindAddons: visualization.spectrum v20.2.0.6 installed
2024-06-30 18:31:22.171 T:893      info <general>: CAddonMgr::FindAddons: webinterface.default v19.x-2.4.8 installed
2024-06-30 18:31:22.171 T:893      info <general>: CAddonMgr::FindAddons: xbmc.addon v21.0.0 installed
2024-06-30 18:31:22.171 T:893      info <general>: CAddonMgr::FindAddons: xbmc.core v0.1.0 installed
2024-06-30 18:31:22.171 T:893      info <general>: CAddonMgr::FindAddons: xbmc.gui v5.17.0 installed
2024-06-30 18:31:22.172 T:893      info <general>: CAddonMgr::FindAddons: xbmc.json v13.5.0 installed
2024-06-30 18:31:22.172 T:893      info <general>: CAddonMgr::FindAddons: xbmc.metadata v2.1.0 installed
2024-06-30 18:31:22.172 T:893      info <general>: CAddonMgr::FindAddons: xbmc.python v3.0.1 installed
2024-06-30 18:31:22.172 T:893      info <general>: CAddonMgr::FindAddons: xbmc.webinterface v1.0.0 installed
2024-06-30 18:31:22.179 T:893      info <general>: Selected Logind/UPower as PowerSyscall
2024-06-30 18:31:22.179 T:893     error <general>: DBus error: org.freedesktop.DBus.Error.ServiceUnknown - The name org.freedesktop.UPower was not provided b
y any .service files
2024-06-30 18:31:22.179 T:893      info <general>: LogindUPowerSyscall - UPower not found, battery information will not be available
2024-06-30 18:31:22.212 T:893      info <general>: CKeyboardLayoutManager: loading keyboard layouts from special://xbmc/system/keyboardlayouts...
2024-06-30 18:31:22.261 T:893   warning <general>: CKeyboardLayoutManager: unable to open special://xbmc/system/keyboardlayouts/persian.xml
2024-06-30 18:31:22.494 T:897      info <general>: Loading special://xbmc/system/Lircmap.xml
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'mceusb'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'XboxDVDDongle'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'Microsoft_Xbox'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'PinnacleSysPCTVRemote'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'anysee'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'iMON-PAD'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'Antec_Veris_RM200'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'MCE_via_iMON'
2024-06-30 18:31:22.508 T:897      info <general>: * Adding remote mapping for device 'TwinHanRemote'
2024-06-30 18:31:22.509 T:897      info <general>: * Adding remote mapping for device 'linux-input-layer'
2024-06-30 18:31:22.509 T:897      info <general>: * Linking remote mapping for 'linux-input-layer' to 'cx23885_remote'
2024-06-30 18:31:22.509 T:897      info <general>: * Adding remote mapping for device 'mediacenter'
2024-06-30 18:31:22.509 T:897      info <general>: * Adding remote mapping for device 'devinput'
2024-06-30 18:31:22.509 T:897      info <general>: * Adding remote mapping for device 'devinput-32'
2024-06-30 18:31:22.509 T:897      info <general>: * Adding remote mapping for device 'devinput-64'
2024-06-30 18:31:22.545 T:893     error <general>: [xkb] couldn't find a Compose file for locale "C" (mapped to "C")
                                                   
2024-06-30 18:31:22.545 T:893   warning <general>: CLibInputKeyboard: Failed to compile localized compose table, composed key support will be disabled
2024-06-30 18:31:22.575 T:893      info <general>: RetroPlayer[RENDER]: Registering renderer factory for DMA
2024-06-30 18:31:22.575 T:893      info <general>: RetroPlayer[RENDER]: Registering renderer factory for OpenGLES
2024-06-30 18:31:22.626 T:893      info <general>: CDRMUtils::FindConnector - using connector: HDMI-A-1
2024-06-30 18:31:22.680 T:893      info <general>: Skipped 1 duplicate messages..
2024-06-30 18:31:22.680 T:893      info <general>: CDRMUtils::FindEncoder - using encoder: 31
2024-06-30 18:31:22.681 T:893      info <general>: CDRMUtils::FindPlanes - using crtc: 89
2024-06-30 18:31:22.681 T:893      info <general>: [display-info] make: 'LG Electronics' model: 'LG TV'
2024-06-30 18:31:22.946 T:893      info <general>: Found resolution 1360x768 with 1360x768 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 59.940063 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080i @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080i @ 59.940063 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 50.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080i @ 50.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 30.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 29.970032 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 25.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 24.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1920x1080 with 1920x1080 @ 23.976025 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1280x720 with 1280x720 @ 61.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1280x720 with 1280x720 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1280x720 with 1280x720 @ 59.940063 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1280x720 with 1280x720 @ 50.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 1024x768 with 1024x768 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 800x600 with 800x600 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 720x576 with 720x576 @ 50.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 720x576 with 720x576i @ 50.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 720x480 with 720x480 @ 59.940063 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 720x480 with 720x480 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 640x480 with 640x480 @ 60.000000 Hz
2024-06-30 18:31:22.947 T:893      info <general>: Skipped 1 duplicate messages..
2024-06-30 18:31:22.947 T:893      info <general>: Found resolution 720x400 with 720x400 @ 70.000000 Hz
2024-06-30 18:31:22.951 T:893      info <general>: EGL_VERSION = 1.4
2024-06-30 18:31:22.951 T:893      info <general>: EGL_VENDOR = Mesa Project
2024-06-30 18:31:22.951 T:893      info <general>: EGL_EXTENSIONS = EGL_ANDROID_blob_cache EGL_ANDROID_native_fence_sync EGL_EXT_buffer_age EGL_EXT_image_dma
_buf_import EGL_EXT_image_dma_buf_import_modifiers EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_context_flush_control EGL_KHR_create_context EGL_KHR_crea
te_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_addresses EGL_KHR_gl_colorspace EGL_KHR_gl_renderbuffer_image EGL_KHR_gl_texture_2D_image EGL_KHR
_gl_texture_cubemap_image EGL_KHR_image EGL_KHR_image_base EGL_KHR_image_pixmap EGL_KHR_no_config_context EGL_KHR_reusable_sync EGL_KHR_surfaceless_context E
GL_EXT_pixel_format_float EGL_KHR_wait_sync EGL_MESA_configless_context EGL_MESA_drm_image EGL_MESA_gl_interop EGL_MESA_image_dma_buf_export EGL_MESA_query_d
river 
2024-06-30 18:31:22.951 T:893      info <general>: EGL_CLIENT_EXTENSIONS = EGL_EXT_client_extensions EGL_EXT_device_base EGL_EXT_device_enumeration EGL_EXT_d
evice_query EGL_EXT_platform_base EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_EXT_platform_device EGL_EXT_explicit_device EGL_MESA_platform_gbm E
GL_KHR_platform_gbm EGL_MESA_platform_surfaceless
2024-06-30 18:31:23.049 T:893      info <general>: CApplication::CreateGUI - using the gbm windowing system
2024-06-30 18:31:23.049 T:893      info <general>: Checking resolution 16
2024-06-30 18:31:23.069 T:893      info <general>: GL_VENDOR = Broadcom
2024-06-30 18:31:23.069 T:893      info <general>: GL_RENDERER = VC4 V3D 2.1
2024-06-30 18:31:23.069 T:893      info <general>: GL_VERSION = OpenGL ES 2.0 Mesa 24.0.5
2024-06-30 18:31:23.069 T:893      info <general>: GL_SHADING_LANGUAGE_VERSION = OpenGL ES GLSL ES 1.0.16
2024-06-30 18:31:23.070 T:893      info <general>: GL_EXTENSIONS = GL_EXT_blend_minmax GL_EXT_multi_draw_arrays GL_EXT_texture_compression_s3tc GL_EXT_textur
e_compression_dxt1 GL_EXT_texture_format_BGRA8888 GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth24 GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OE
S_mapbuffer GL_OES_rgb8_rgba8 GL_OES_stencil8 GL_OES_texture_npot GL_OES_vertex_half_float GL_OES_EGL_image GL_OES_depth_texture GL_AMD_performance_monitor G
L_OES_packed_depth_stencil GL_OES_get_program_binary GL_APPLE_texture_max_level GL_EXT_discard_framebuffer GL_EXT_read_format_bgra GL_NV_pack_subimage GL_NV_
texture_barrier GL_EXT_frag_depth GL_NV_fbo_color_attachments GL_OES_EGL_image_external GL_OES_EGL_sync GL_OES_vertex_array_object GL_ANGLE_pack_reverse_row_
order GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_EXT_unpack_subimage GL_NV_draw_buffers GL_NV_read_buffer GL_NV_read_depth GL_NV_
read_depth_stencil GL_NV_read_stencil GL_APPLE_sync GL_EXT_draw_buffers GL_EXT_map_buffer_range GL_KHR_debug GL_KHR_texture_compression_astc_ldr GL_NV_genera
te_mipmap_sRGB GL_NV_pixel_buffer_object GL_OES_required_internalformat GL_OES_surfaceless_context GL_EXT_debug_label GL_EXT_separate_shader_objects GL_EXT_c
ompressed_ETC1_RGB8_sub_texture GL_EXT_draw_elements_base_vertex GL_EXT_texture_border_clamp GL_KHR_context_flush_control GL_OES_draw_elements_base_vertex GL
_OES_texture_border_clamp GL_KHR_no_error GL_KHR_texture_compression_astc_sliced_3d GL_EXT_texture_compression_s3tc_srgb GL_KHR_parallel_shader_compile GL_ME
SA_tile_raster_order GL_MESA_sampler_objects GL_MESA_bgra 
2024-06-30 18:31:23.216 T:893      info <general>: GLES: Maximum texture width: 2048
2024-06-30 18:31:24.037 T:893      info <general>: load keymapping
2024-06-30 18:31:24.042 T:893      info <general>: Loading special://xbmc/system/keymaps/appcommand.xml
2024-06-30 18:31:24.043 T:893      info <general>: Loading special://xbmc/system/keymaps/customcontroller.AppleRemote.xml
2024-06-30 18:31:24.045 T:893      info <general>: Loading special://xbmc/system/keymaps/customcontroller.Harmony.xml
2024-06-30 18:31:24.047 T:893      info <general>: Loading special://xbmc/system/keymaps/customcontroller.SiriRemote.xml
2024-06-30 18:31:24.049 T:893      info <general>: Loading special://xbmc/system/keymaps/gamepad.xml
2024-06-30 18:31:24.052 T:893      info <general>: Loading special://xbmc/system/keymaps/joystick.xml
2024-06-30 18:31:24.055 T:893      info <general>: Loading special://xbmc/system/keymaps/keyboard.xml
2024-06-30 18:31:24.061 T:893      info <general>: Loading special://xbmc/system/keymaps/mouse.xml
2024-06-30 18:31:24.063 T:893      info <general>: Loading special://xbmc/system/keymaps/remote.xml
2024-06-30 18:31:24.067 T:893      info <general>: Loading special://xbmc/system/keymaps/touchscreen.xml
2024-06-30 18:31:24.071 T:893      info <general>: GUI format 1360x768, Display 1360x768 @ 60.000000 Hz
2024-06-30 18:31:24.108 T:899      info <general>: CAESinkALSA - Unable to open device "surround71" for playback
2024-06-30 18:31:24.109 T:899      info <general>: CAESinkALSA - Unable to open device "surround51" for playback
2024-06-30 18:31:24.109 T:899      info <general>: CAESinkALSA - Unable to open device "surround71" for playback
2024-06-30 18:31:24.110 T:899      info <general>: CAESinkALSA - Unable to open device "surround40" for playback
2024-06-30 18:31:24.110 T:899      info <general>: CAESinkALSA - Unable to open device "surround51" for playback
2024-06-30 18:31:24.110 T:899      info <general>: CAESinkALSA - Unable to open device "surround71" for playback
2024-06-30 18:31:24.140 T:899   warning <general>: Pulseaudio module module-allow-passthrough not loaded - opening PT devices might fail
2024-06-30 18:31:24.141 T:899      info <general>: Found 2 Lists of Devices
2024-06-30 18:31:24.141 T:899      info <general>: Enumerated ALSA devices:
2024-06-30 18:31:24.141 T:899      info <general>:     Device 1
2024-06-30 18:31:24.141 T:899      info <general>:         m_deviceName      : @
2024-06-30 18:31:24.141 T:899      info <general>:         m_displayName     : Default (vc4-hdmi MAI PCM i2s-hifi-0) ()
2024-06-30 18:31:24.141 T:899      info <general>:         m_displayNameExtra: 
2024-06-30 18:31:24.141 T:899      info <general>:         m_deviceType      : AE_DEVTYPE_PCM
2024-06-30 18:31:24.141 T:899      info <general>:         m_channels        : FL, FR
2024-06-30 18:31:24.141 T:899      info <general>:         m_sampleRates     : 32000,44100,48000,96000,192000
2024-06-30 18:31:24.141 T:899      info <general>:         m_dataFormats     : AE_FMT_S24NE3,AE_FMT_S24NE4,AE_FMT_S32NE,AE_FMT_S16NE,AE_FMT_S16LE,AE_FMT_S16B
E
2024-06-30 18:31:24.141 T:899      info <general>:         m_streamTypes     : No passthrough capabilities
2024-06-30 18:31:24.141 T:899      info <general>:     Device 2
2024-06-30 18:31:24.141 T:899      info <general>:         m_deviceName      : sysdefault
2024-06-30 18:31:24.141 T:899      info <general>:         m_displayName     : vc4-hdmi ()
2024-06-30 18:31:24.141 T:899      info <general>:         m_displayNameExtra: MAI PCM i2s-hifi-0
2024-06-30 18:31:24.141 T:899      info <general>:         m_deviceType      : AE_DEVTYPE_PCM
2024-06-30 18:31:24.141 T:899      info <general>:         m_channels        : FL, FR
2024-06-30 18:31:24.141 T:899      info <general>:         m_sampleRates     : 32000,44100,48000,96000,192000
2024-06-30 18:31:24.141 T:899      info <general>:         m_dataFormats     : AE_FMT_S24NE3,AE_FMT_S24NE4,AE_FMT_S32NE,AE_FMT_S16NE,AE_FMT_S16LE,AE_FMT_S16B
E
2024-06-30 18:31:24.142 T:899      info <general>:         m_streamTypes     : No passthrough capabilities
2024-06-30 18:31:24.142 T:899      info <general>:     Device 3
2024-06-30 18:31:24.142 T:899      info <general>:         m_deviceName      : sysdefault:CARD=vc4hdmi
2024-06-30 18:31:24.142 T:899      info <general>:         m_displayName     : vc4-hdmi (vc4hdmi)
2024-06-30 18:31:24.142 T:899      info <general>:         m_displayNameExtra: MAI PCM i2s-hifi-0
2024-06-30 18:31:24.142 T:899      info <general>:         m_deviceType      : AE_DEVTYPE_PCM
2024-06-30 18:31:24.142 T:899      info <general>:         m_channels        : FL, FR
2024-06-30 18:31:24.142 T:899      info <general>:         m_sampleRates     : 32000,44100,48000,96000,192000
2024-06-30 18:31:24.142 T:899      info <general>:         m_dataFormats     : AE_FMT_S24NE3,AE_FMT_S24NE4,AE_FMT_S32NE,AE_FMT_S16NE,AE_FMT_S16LE,AE_FMT_S16B
E
2024-06-30 18:31:24.142 T:899      info <general>:         m_streamTypes     : No passthrough capabilities
2024-06-30 18:31:24.142 T:899      info <general>:     Device 4
2024-06-30 18:31:24.142 T:899      info <general>:         m_deviceName      : hdmi:CARD=vc4hdmi,DEV=0
2024-06-30 18:31:24.142 T:899      info <general>:         m_displayName     : vc4-hdmi (vc4hdmi)
2024-06-30 18:31:24.142 T:899      info <general>:         m_displayNameExtra: GSM LG TV on HDMI
2024-06-30 18:31:24.142 T:899      info <general>:         m_deviceType      : AE_DEVTYPE_HDMI
2024-06-30 18:31:24.142 T:899      info <general>:         m_channels        : FL, FR
2024-06-30 18:31:24.142 T:899      info <general>:         m_sampleRates     : 32000,44100,48000,96000,192000
2024-06-30 18:31:24.142 T:899      info <general>:         m_dataFormats     : AE_FMT_RAW,AE_FMT_S24NE3,AE_FMT_S24NE4,AE_FMT_S32NE,AE_FMT_S16NE,AE_FMT_S16LE,
AE_FMT_S16BE,AE_FMT_U8,AE_FMT_RAW
2024-06-30 18:31:24.142 T:899      info <general>:         m_streamTypes     : STREAM_TYPE_AC3,STREAM_TYPE_DTSHD,STREAM_TYPE_DTSHD_MA,STREAM_TYPE_DTSHD_CORE,
STREAM_TYPE_DTS_1024,STREAM_TYPE_DTS_2048,STREAM_TYPE_DTS_512,STREAM_TYPE_EAC3,STREAM_TYPE_TRUEHD
2024-06-30 18:31:24.142 T:899      info <general>: Enumerated PULSE devices:
2024-06-30 18:31:24.142 T:899      info <general>:     Device 1
2024-06-30 18:31:24.142 T:899      info <general>:         m_deviceName      : Default
2024-06-30 18:31:24.142 T:899      info <general>:         m_displayName     : Default
2024-06-30 18:31:24.142 T:899      info <general>:         m_displayNameExtra: Bluetooth Audio (PULSEAUDIO)
2024-06-30 18:31:24.142 T:899      info <general>:         m_deviceType      : AE_DEVTYPE_PCM
2024-06-30 18:31:24.142 T:899      info <general>:         m_channels        : FL, FR
2024-06-30 18:31:24.142 T:899      info <general>:         m_sampleRates     : 5512,8000,11025,16000,22050,32000,44100,48000,64000,88200,96000,176400,192000,
384000
2024-06-30 18:31:24.142 T:899      info <general>:         m_dataFormats     : AE_FMT_U8,AE_FMT_S16NE,AE_FMT_S24NE3,AE_FMT_S24NE4,AE_FMT_S32NE,AE_FMT_FLOAT
2024-06-30 18:31:24.142 T:899      info <general>:         m_streamTypes     : No passthrough capabilities
2024-06-30 18:31:24.142 T:900      info <general>: CActiveAESink::OpenSink - initialize sink
2024-06-30 18:31:24.142 T:900      info <general>: CAESinkALSA::Initialize - Attempting to open device "hdmi:CARD=vc4hdmi,DEV=0"
2024-06-30 18:31:24.147 T:900      info <general>: CAESinkALSA::Initialize - Opened device "hdmi:CARD=vc4hdmi,DEV=0,AES0=0x04,AES1=0x82,AES2=0x00,AES3=0x00"
2024-06-30 18:31:24.148 T:900      info <general>: CAESinkALSA::InitializeHW - Your hardware does not support AE_FMT_FLOAT, trying other formats
2024-06-30 18:31:24.148 T:900      info <general>: CAESinkALSA::InitializeHW - Using data format AE_FMT_S24NE3
2024-06-30 18:31:24.153 T:893      info <general>: CLangInfo: loading resource.language.en_gb language information...
2024-06-30 18:31:24.166 T:893      info <general>: global locale set to C
2024-06-30 18:31:24.166 T:893      info <general>: CLangInfo: loading resource.language.en_gb language strings...
2024-06-30 18:31:24.326 T:903      info <general>: Running database version Addons33
2024-06-30 18:31:24.330 T:903      info <general>: Running database version ViewModes6
2024-06-30 18:31:24.349 T:903      info <general>: Running database version Textures13
2024-06-30 18:31:24.366 T:903      info <general>: Running database version MyMusic83
2024-06-30 18:31:24.397 T:903      info <general>: Running database version MyVideos131
2024-06-30 18:31:24.404 T:903      info <general>: Running database version TV44
2024-06-30 18:31:24.410 T:903      info <general>: Running database version Epg16
2024-06-30 18:31:24.521 T:893      info <general>: Unloaded skin
2024-06-30 18:31:24.524 T:893      info <general>:   load skin from: /usr/share/kodi/addons/skin.estuary/ (version: 4.0.0)
2024-06-30 18:31:24.524 T:893      info <general>:   load fonts for skin...
2024-06-30 18:31:24.553 T:893      info <general>: Loading skin includes from /usr/share/kodi/addons/skin.estuary/xml/Includes.xml
2024-06-30 18:31:24.733 T:893      info <general>: LoadFontsFromFile: Loading <fontset> with name 'Default' from '/usr/share/kodi/addons/skin.estuary/xml/Fon
t.xml'
2024-06-30 18:31:24.855 T:893      info <general>: LoadTimers: Trying to load skin timers from /usr/share/kodi/addons/skin.estuary/xml/Timers.xml
2024-06-30 18:31:24.866 T:893      info <general>:   load new skin...
2024-06-30 18:31:24.866 T:893      info <general>: Loading custom window XMLs from skin path /usr/share/kodi/addons/skin.estuary/xml
2024-06-30 18:31:24.898 T:893      info <general>:   initialize new skin...
2024-06-30 18:31:24.898 T:893      info <general>: Loading skin file: Custom_1109_TopBarOverlay.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.957 T:893      info <general>: Loading skin file: DialogBusy.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.960 T:893      info <general>: Loading skin file: DialogNotification.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.962 T:893      info <general>: Loading skin file: DialogSeekBar.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.989 T:893      info <general>: Loading skin file: DialogExtendedProgressBar.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.993 T:893      info <general>: Loading skin file: Pointer.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.995 T:893      info <general>: Loading skin file: DialogBusy.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.997 T:893      info <general>: Loading skin file: DialogVolumeBar.xml, load type: LOAD_ON_GUI_INIT
2024-06-30 18:31:24.999 T:893      info <general>: Loading resource://resource.uisounds.kodi/sounds.xml
2024-06-30 18:31:25.091 T:893      info <general>:   skin loaded...
2024-06-30 18:31:25.092 T:893      info <general>: Loading skin file: Startup.xml, load type: LOAD_EVERY_TIME
2024-06-30 18:31:25.103 T:893      info <general>: Loading skin file: Home.xml, load type: KEEP_IN_MEMORY
2024-06-30 18:31:25.290 T:893   warning <general>: JSONRPC: Could not parse type "Setting.Details.SettingList"
2024-06-30 18:31:25.290 T:893      info <general>: JSONRPC: Adding type "Setting.Details.SettingList" to list of incomplete definitions (waiting for "Setting
.Details.Setting")
2024-06-30 18:31:25.291 T:893      info <general>: JSONRPC: Resolving incomplete types/methods referencing Setting.Details.Setting
2024-06-30 18:31:25.332 T:893      info <general>: JSONRPC v13.5.0: Successfully initialized
2024-06-30 18:31:25.333 T:893      info <general>: [Media Detection] starting service for optical media detection
2024-06-30 18:31:25.378 T:893      info <general>: AddOnLog: peripheral.joystick: Enabling joystick interface "linux"
2024-06-30 18:31:25.383 T:893      info <general>: Register - new keyboard device registered on application->keyboard: Keyboard (0000:0000)
2024-06-30 18:31:25.388 T:893      info <general>: Register - new mouse device registered on application->mouse: Mouse (0000:0000)
2024-06-30 18:31:25.389 T:893      info <general>: Loading player core factory settings from special://xbmc/system/playercorefactory.xml.
2024-06-30 18:31:25.391 T:893      info <general>: Loaded playercorefactory configuration
2024-06-30 18:31:25.392 T:893      info <general>: Loading player core factory settings from special://masterprofile/playercorefactory.xml.
2024-06-30 18:31:25.392 T:893      info <general>: special://masterprofile/playercorefactory.xml does not exist. Skipping.
2024-06-30 18:31:25.392 T:893      info <general>: removing tempfiles
2024-06-30 18:31:25.408 T:893      info <general>: initialize done
2024-06-30 18:31:25.408 T:893      info <general>: Running the application...
2024-06-30 18:31:25.427 T:893      info <general>: starting zeroconf publishing
2024-06-30 18:31:25.427 T:893      info <general>: starting upnp client
2024-06-30 18:31:25.433 T:893      info <general>: starting upnp renderer
2024-06-30 18:31:25.461 T:924      info <general>: ES: Starting UDP Event server on port 9777
2024-06-30 18:31:25.461 T:924      info <general>: UDP: Listening on port 9777 (ipv6 : false)
2024-06-30 18:31:25.462 T:893      info <general>: JSONRPC Server: Successfully initialized
2024-06-30 18:31:25.509 T:909      info <general>: Register - new cec device registered on cec->Linux: CEC Adapter (0000:0000)
2024-06-30 18:31:29.622 T:915      info <general>: SETTINGS: <module> # ## LibreELEC Addon ## 12.0
2024-06-30 18:31:29.864 T:915      info <general>: SETTINGS: set_keyboard_layout # Settings keyboard layout: us
2024-06-30 18:31:29.871 T:915      info <general>: SETTINGS: set_keyboard_layout # Found keyboard layout: /usr/lib/keymaps/us/us.bmap
2024-06-30 18:31:29.871 T:915      info <general>: SETTINGS: set_keyboard_layout # Executing loadkmap < /usr/lib/keymaps/us/us.bmap
2024-06-30 18:31:32.090 T:915      info <general>: SETTINGS: __init__ # updateThread Started
2024-06-30 18:31:32.092 T:915      info <general>: SETTINGS: set_auto_update # auto
2024-06-30 18:31:32.311 T:989      info <general>: SETTINGS: run # Waiting
```

**Starting and stopping of first video:**
```
2024-06-30 18:32:24.209 T:893      info <general>: VideoPlayer::OpenFile: http://<IP OF MY VIDEOSERVER>:5000/webman/3rdparty/VideoStation/controller/ui/cgi/vtestreaming.cgi/eyJpZCI6InVwbnA6dXVpZDo3NzU3OGIyOS0zZmFkLWI5NTctYzM4MC1hYzgxYmZlMmIxNTIiLCJtZXRob2QiOiJvcGVuIiwidG9rZW4iOiJ4VGFRQkF2S3pwZENfMTcxOTc2NTE0NCJ9Cg==.mp4
2024-06-30 18:32:24.363 T:1010     info <general>: Creating InputStream
2024-06-30 18:32:24.819 T:1010     info <general>: Creating Demuxer
2024-06-30 18:32:25.757 T:1010     info <general>: Opening stream: 0 source: 256
2024-06-30 18:32:25.757 T:1010     info <general>: Creating video codec with codec id: 27
2024-06-30 18:32:25.757 T:1010     info <general>: CDVDVideoCodecDRMPRIME::Open - using decoder V4L2 mem2mem H.264 decoder wrapper
2024-06-30 18:32:25.794 T:1010     info <general>: Creating video thread
2024-06-30 18:32:25.794 T:1017     info <general>: running thread: video_thread
2024-06-30 18:32:25.795 T:1010     info <general>: Opening stream: 1 source: 256
2024-06-30 18:32:25.795 T:1010     info <general>: Finding audio codec for: 86018
2024-06-30 18:32:25.797 T:1010     info <general>: CDVDAudioCodecFFmpeg::Open() Successful opened audio decoder aac
2024-06-30 18:32:25.797 T:1010     info <general>: OpenStream: Allowing max Out-Of-Sync Value of 10 ms
2024-06-30 18:32:25.798 T:1010     info <general>: Creating audio thread
2024-06-30 18:32:25.798 T:1018     info <general>: running thread: CVideoPlayerAudio::Process()
2024-06-30 18:32:25.809 T:1018     info <general>: Creating audio stream (codec id: 86018, channels: 2, sample rate: 48000, no pass-through)
2024-06-30 18:32:25.856 T:900      info <general>: CActiveAESink::OpenSink - initialize sink
2024-06-30 18:32:26.054 T:900      info <general>: CAESinkALSA::Initialize - Attempting to open device "hdmi:CARD=vc4hdmi,DEV=0"
2024-06-30 18:32:26.063 T:900      info <general>: CAESinkALSA::Initialize - Opened device "hdmi:CARD=vc4hdmi,DEV=0,AES0=0x04,AES1=0x82,AES2=0x00,AES3=0x02"
2024-06-30 18:32:26.064 T:900      info <general>: CAESinkALSA::InitializeHW - Your hardware does not support AE_FMT_FLOAT, trying other formats
2024-06-30 18:32:26.065 T:900      info <general>: CAESinkALSA::InitializeHW - Using data format AE_FMT_S24NE3
2024-06-30 18:32:26.158 T:893      info <general>: Loading skin file: VideoFullScreen.xml, load type: KEEP_IN_MEMORY
2024-06-30 18:32:46.815 T:893      info <general>: CVideoPlayer::CloseFile()
2024-06-30 18:32:46.815 T:893      info <general>: VideoPlayer: waiting for threads to exit
2024-06-30 18:32:46.823 T:1010     info <general>: CVideoPlayer::OnExit()
2024-06-30 18:32:46.823 T:1010     info <general>: Closing stream player 1
2024-06-30 18:32:46.823 T:1010     info <general>: Waiting for audio thread to exit
2024-06-30 18:32:46.824 T:1018     info <general>: thread end: CVideoPlayerAudio::OnExit()
2024-06-30 18:32:46.825 T:1010     info <general>: Closing audio device
2024-06-30 18:32:46.876 T:1010     info <general>: Deleting audio codec
2024-06-30 18:32:46.876 T:1010     info <general>: Closing stream player 2
2024-06-30 18:32:46.876 T:1010     info <general>: waiting for video thread to exit
2024-06-30 18:32:46.876 T:900      info <general>: CActiveAESink::OpenSink - initialize sink
2024-06-30 18:32:46.918 T:1017  warning <general>: OutputPicture - timeout waiting for buffer
2024-06-30 18:32:46.919 T:1017     info <general>: thread end: video_thread
2024-06-30 18:32:46.922 T:1010     info <general>: deleting video codec
2024-06-30 18:32:46.958 T:893      info <general>: VideoPlayer: finished waiting
2024-06-30 18:32:46.959 T:893      info <general>: CVideoPlayer::CloseFile()
2024-06-30 18:32:46.959 T:893      info <general>: VideoPlayer: waiting for threads to exit
2024-06-30 18:32:46.959 T:893      info <general>: VideoPlayer: finished waiting
2024-06-30 18:32:47.072 T:900      info <general>: CAESinkALSA::Initialize - Attempting to open device "hdmi:CARD=vc4hdmi,DEV=0"
2024-06-30 18:32:47.077 T:900      info <general>: CAESinkALSA::Initialize - Opened device "hdmi:CARD=vc4hdmi,DEV=0,AES0=0x04,AES1=0x82,AES2=0x00,AES3=0x00"
2024-06-30 18:32:47.078 T:900      info <general>: CAESinkALSA::InitializeHW - Your hardware does not support AE_FMT_FLOAT, trying other formats
2024-06-30 18:32:47.078 T:900      info <general>: CAESinkALSA::InitializeHW - Using data format AE_FMT_S24NE3
```
**Starting of 2nd video until the error appears:**
```
2024-06-30 18:32:55.477 T:893      info <general>: VideoPlayer::OpenFile: http://IP OF MY VIDEOSERVER>:5000/webman/3rdparty/VideoStation/controller/ui/cgi/vtestreaming.cgi/eyJpZCI6InVwbnA6dXVpZDo3NzU3OGIyOS0zZmFkLWI5NTctYzM4MC1hYzgxYmZlMmIxNTIiLCJtZXRob2QiOiJvcGVuIiwidG9rZW4iOiJON2lJNWRyVUg3WTVfMTcxOTc2NTE3NSJ9Cg==.mp4
2024-06-30 18:32:55.614 T:1093     info <general>: Creating InputStream
2024-06-30 18:32:55.782 T:1093    error <general>: CVideoPlayer::OpenInputStream - error opening [http://IP OF MY VIDEOSERVER>:5000/webman/3rdparty/VideoStation/controller/ui/cgi/vtestreaming.cgi/eyJpZCI6InVwbnA6dXVpZDo3NzU3OGIyOS0zZmFkLWI5NTctYzM4MC1hYzgxYmZlMmIxNTIiLCJtZXRob2QiOiJvcGVuIiwidG9rZW4iOiJON2lJNWRyVUg3WTVfMTcxOTc2NTE3NSJ9Cg==.mp4]
2024-06-30 18:32:55.783 T:1093     info <general>: CVideoPlayer::OnExit()
2024-06-30 18:32:55.799 T:893      info <general>: Loading skin file: DialogConfirm.xml, load type: KEEP_IN_MEMORY
2024-06-30 18:32:55.836 T:893      info <general>: CVideoPlayer::CloseFile()
2024-06-30 18:32:55.837 T:893      info <general>: VideoPlayer: waiting for threads to exit
2024-06-30 18:32:55.837 T:893      info <general>: VideoPlayer: finished waiting
2024-06-30 18:32:56.368 T:893      info <general>: VideoPlayer::OpenFile: http://IP OF MY VIDEOSERVER>:5000/webman/3rdparty/VideoStation/controller/ui/cgi/vtestreaming.cgi/eyJpZCI6InVwbnA6dXVpZDo3NzU3OGIyOS0zZmFkLWI5NTctYzM4MC1hYzgxYmZlMmIxNTIiLCJtZXRob2QiOiJvcGVuIiwidG9rZW4iOiJON2lJNWRyVUg3WTVfMTcxOTc2NTE3NSJ9Cg==.mp4
2024-06-30 18:32:56.534 T:1098     info <general>: Creating InputStream
2024-06-30 18:32:56.666 T:1098    error <general>: CVideoPlayer::OpenInputStream - error opening [http://IP OF MY VIDEOSERVER>:5000/webman/3rdparty/VideoStation/controller/ui/cgi/vtestreaming.cgi/eyJpZCI6InVwbnA6dXVpZDo3NzU3OGIyOS0zZmFkLWI5NTctYzM4MC1hYzgxYmZlMmIxNTIiLCJtZXRob2QiOiJvcGVuIiwidG9rZW4iOiJON2lJNWRyVUg3WTVfMTcxOTc2NTE3NSJ9Cg==.mp4]
2024-06-30 18:32:56.666 T:1098     info <general>: CVideoPlayer::OnExit()
2024-06-30 18:32:56.698 T:893      info <general>: CVideoPlayer::CloseFile()
2024-06-30 18:32:56.698 T:893      info <general>: VideoPlayer: waiting for threads to exit
2024-06-30 18:32:56.698 T:893      info <general>: VideoPlayer: finished waiting
```
If I open the URL in a browser is is starting the expected video.

**After confirmation of the error message**
```
2024-06-30 18:33:57.982 T:893      info <general>: Loading controller layout: /usr/share/kodi/addons/game.controller.keyboard/resources/layout.xml
2024-06-30 18:33:58.083 T:900      info <general>: CActiveAESink::OpenSink - initialize sink
2024-06-30 18:33:58.083 T:900      info <general>: CAESinkALSA::Initialize - Attempting to open device "hdmi:CARD=vc4hdmi,DEV=0"
2024-06-30 18:33:58.092 T:900      info <general>: CAESinkALSA::Initialize - Opened device "hdmi:CARD=vc4hdmi,DEV=0,AES0=0x04,AES1=0x82,AES2=0x00,AES3=0x00"
2024-06-30 18:33:58.093 T:900      info <general>: CAESinkALSA::InitializeHW - Your hardware does not support AE_FMT_FLOAT, trying other formats
2024-06-30 18:33:58.093 T:900      info <general>: CAESinkALSA::InitializeHW - Using data format AE_FMT_S24NE3
```

### Screenshots 
Here are some links or screenshots to help explain the problem:

![image](https://github.com/xbmc/xbmc/assets/29121472/4aa3cb63-55b9-4f22-94ae-9e1a4fcc91da)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [X] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP
 - [X] LibreELEC

 - Operating system version/name: Debian 12.5, LibreELEC 12
 - Kodi version: 20.5 and 21

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by arminfuerst** [comment]
@sarbes Could you please explain which rule I was ignoring so I can provide all information you need?

**Comment by arminfuerst** [comment]
I just saw, my log seems to be created with disabled debug mode, I just created an additional log and pasted it to
[paste.kodi.tv](https://paste.kodi.tv/usowiwawic.kodi). Does this unblock the bug?
I made following changes to the log:
- I changed the username of the local machine to "USERNAME"
- I changed the name of the affected kodi device (the one used to create the log) to AFFECTEDKODIDEVICE
- I changed the IP-address of the synology server with SYNOLOGYSERVER
- I changed the IP-addresses of other active kodi devices in the network to "KODIDEVICE1" and "KODIDEVICE2"

The line with timestamp "2024-08-22 20:21:08.506" was the last line after the video was played for the first time (successfully).
The line with timestamp "2024-08-22 20:21:13.600" was just before the second attempt (displaying the error documented above)

**Comment by anohren** [comment]
I came here to file a report of this exact bug (which I have experienced for over a year on multiple platforms ever since updating to a version > 19, and have seen described a long time ago on the Kodi forum, with zero replies) but if this report only resulted in a "rules ignored" tag then I think my chances of spotting what the problem is and avoiding doing the same thing are slim to none. I looked at the issue template but couldn't see it off the top of my head. I assume there were no duplicates.

For now I'll refrain from posting my log in this issue lest I accidentally break some rule by doing that (?) Well, I don't know...

**Comment by basilgello** [comment]
@anohren Feel free to post :) You are protected!

**Comment by arminfuerst** [comment]
@basilgello I tried to fix the reason for "Ignored rules", but there was no change to this issue after I uploaded more detailed logs. I'd be happy to provide additional details - if I know what helps to find this issue.

**Comment by the-black-eagle** [comment]
@arminfuerst You need to put kodi into debug mode and then reproduce your issue.  Upload the resulting FULL log to a paste site and link back to that here.  DO NOT post part of the log here, or cut or edit the log.  In spite of what you might think, nobody can do anything with your local IP addresses / machine names.  

Notice that the snippet you posted does not contain the word DEBUG anywhere!

Thanks.

**Comment by arminfuerst** [comment]
> @arminfuerst You need to put kodi into debug mode and then reproduce your issue. Upload the resulting FULL log to a paste site and link back to that here. DO NOT post part of the log here, or cut or edit the log. In spite of what you might think, nobody can do anything with your local IP addresses / machine names.
> 
> Notice that the snippet you posted does not contain the word DEBUG anywhere!
> 
> Thanks.

That's - what I thought to have added on August 22nd?

**Comment by anohren** [comment]
> @anohren Feel free to post :) You are protected!

😂

Oh Vasyl... hope you're doing well. In my attempts to revive an old machine I distinctly remember bugging you on email about which GL version Kodi from Debian actually requires, and us coming to the conclusion that it was...undefined. Ironically the only Kodi installation I have now that consistently always accepts remote DLNA/UPnP playback (i.e. doesn't exhibit this issue) is the one running on that old machine: 17.6 

I was just a bit demotivated here because I couldn't really open a separate issue without creating a duplicate (which I'm asked not to do in the how-to) and not knowing whether this issue would receive any more attention. Here goes:

The debug log of this issue running on macOS 10.14.6:
https://paste.kodi.tv/ogirisafic.kodi

### Description
For me, this problem started appearing on Kodi 20 on **OSMC** after updating from 19, and continued after updating to 21. On version 20 on OSMC I recall being able to "send" one video using DLNA to play on Kodi, and every subsequent attempt failing. Now my experience is that most if not all attempts are failing, including the first attempt. The first attempt is certainly failinig on Kodi 21 on **macOS** (shown in the log: `http://192.168.0.201:1314/%25/Files/Shared/WebVideo10.mp4`). I attribute these behaviors to the same underlying problem, introduced in version 20 (but that's an assumption). 

### How to reproduce
This is how I always manage to reproduce the problem:

1. Download a SD youtube video to iPhone using the iOS app PlayerXtreme
2. Open that file (by sharing it using the share-sheet) in the iOS app TV Assist (a DLNA server/remote app -- no purchase required)
3. Try to play that file on Kodi 21.0 on macOS (in this case macOS 10.14.6)
4. Kodi message box appears with localized equivalent of "one or more items failed to play [...]"

The exact same file can be played in the exact same fashion on Kodi 17 as well as on a Samsung TV (built-in DLNA functionality) on the same network -- and these players never ever fail to play these files -- leading me to believe it might be a Kodi problem.

**Comment by enen92** [comment]
Can't reproduce this in Kodi Piers unfortunately (Kodi on android + bubbleupnp on the phone)

**Comment by arminfuerst** [comment]
@enen92 I just compiled kodi from the master branch and can reproduce the error. I pasted the log to https://paste.kodi.tv/oguhasanac.kodi
Does this help?

**Comment by anohren** [comment]
I can reproduce it on any machine (ARM64, i386, amd64) that I install Kodi 21 on — Raspberry Pi (OSMC), Samsung netbook (Debian), MacBook Air (macOS)... Though I applaud @arminfuerst for jumping through the hoop of reproducing it on a more recent non-release version. 

I can only speculate that it's a combination of certain DLNA control points (terminology?) + servers (the app I use acts as both as I understand it) + Kodi version >= 20 that makes the problem manifest. That's my experience at least, and is in line with @arminfuerst's description of upgrading from v19 → v21. 

The one server that I know for a fact exhibits the problem is TV Assist on iOS, which I'm assuming is more readily available than the Synology Video Station that @arminfuerst is using. As I remember I have previously been in the situation where I could send videos between Kodi instances using DLNA/UPnP while at the same time receiving this error from other DLNA sources (e.g. the iOS app).

**Comment by basilgello** [comment]
Would be great to capture UPnP traffic with dumpcap / wireshark - because Armin's log shows nothing at a first glance that immediately reveals the problem

**Comment by arminfuerst** [comment]
@basilgello I'll try to capture a log, are the standard capture settings of wireshark fine or would you need additional details?
As I could reproduce it with the current master branch: If I shall add some log infos in the source, this would also be possible. If you tell me, what to add or send me a patch, I could recompile and add additional logs.
And could someone please remove the "Ignored rules" tag, I think it's misleading.

**Comment by basilgello** [comment]
Yes, standard wireshark is fine and in order to keep your privacy, you can send me the log file attached to basilgello@kodi.tv

**Comment by arminfuerst** [comment]
> Yes, standard wireshark is fine and in order to keep your privacy, you can send me the log file attached to [basilgello@kodi.tv](mailto:basilgello@kodi.tv)

I prepared some captures, starting with the DLNA-registration done when Kodi starts, first playback (successful) and second playback (with the error). As soon as you fetched the password (see my email), I'll provide the ZIP.
In the ZIP you'll find following files:
- DLNA-full-log.pcapng - this is a single file with all three actions
- DLNA-01-registration.pcapng - this is just the registration process
- DLNA-02-first-start.pcapng - this is the first playback (successful)
- DLNA-03-second-start.pcapng - this is the second playback (with the error)

These are separate captures, meaning I reproduced the error twice to create those captures. The captures itself are unchanged.

**Comment by basilgello** [comment]
Great! I received the password!

**Comment by arminfuerst** [comment]
I just sent the file as email.

**Comment by arminfuerst** [comment]
@basilgello they are on the way. Just one thought: I can imagine, the issue is not caused during the start of the video but when the first videos ends. I think of the possibility of a missing cleanup effecting the next start.

**Comment by anohren** [comment]
> not caused during the start of the video but when the first videos ends

If so then I guess I'll have to start to capture as well since for me it fails immediately on first video.

**Comment by basilgello** [comment]
The new debug log shows failure likely caused by inability to [open URL](https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamFile.cpp#L66) Debugging CFile is tedious task, but lets try :)

**Comment by basilgello** [comment]
With Armin's help I managed to find the flawed part: https://github.com/xbmc/xbmc/blame/master/xbmc/filesystem/File.cpp#L273 But who knows that caching part now?

**Comment by anohren** [comment]
Good work you two. 

I suppose that now the problem is what part of the control flow changed between v19 and v20 to make this 15 year old line start affecting UPnP? Or could this potentially be more wide-spread than just UPnP streams?

**Comment by basilgello** [comment]
Potentially more. But your UPnP issue is probably a separate one because you have no video playing in the first place

**Comment by arminfuerst** [comment]
@anohren Do you have a chance of trying to reproduce on an Andoid-compatible device? If yes, I could try whether I can reproduce the behavior observed by you and whether the fix would also work for your situation.

**Comment by anohren** [comment]
@basilgello Right, that's a bummer. 

@arminfuerst I can't think of a way at the moment. In case I missed something here: weren't you experiencing this on LibreELEC? I couldn't find where you explicitly say which device you were using, but BCM2835 and the host name suggests Raspberry Pi. Where does Android enter the picture? I have a few Pis, but that's as close as it gets I'm afraid. 

Also, what is this "fix" you speak of?



I'll see if I can capture some useful traffic and then we can decide whether I should create a separate issue.

**Comment by arminfuerst** [comment]
@anohren I experienced this on LibeELEC and could reproduce it directly on Debian. But the behavior was identical on all devices, it was always starting the first video, but no subsequent one. You experienced not even starting a single video, and if I could reproduce this, I could verify whether this has the same cause or it's a different issue.
The "fix" would just be the quick fix (you might also call it "hack") from yesterday to ensure that this is the only line in the code causing the issue.

**Comment by basilgello** [comment]
That would be indeed the "hack". Correct way to go is to see why the last return statement fails in `CDirectoryCache::FileExists` while the cached representation is in the list.

Maybe you can help me with one more Rustdesk session tonight @arminfuerst ?

**Comment by arminfuerst** [comment]
@basilgello Another rustdesk session this evening is possible, what time would be fine for you? We can arrange the details also via email, if this would be suitable to you, I don't think our appointments are relevant for this issue ;)

**Comment by basilgello** [comment]
Definitely! Email is better :)

**Comment by basilgello** [comment]
So, key takeaways:

1. On first video playback after Kodi startup, `g_directoryCache` has no HTTP "directory" inside and `FileExists` returns `false` but `bPathInCache` is also `false` thus video file plays

2. Subtitle scanner attempts to scan http directory with no listing, sees 0 items inside and caches it once

3. On subsequent video run, the file name is expected to be in cache of the directory that subtitle scanner previously populated via `CDirectory::GetDirectory`. The video file path appears not to be in that cache and open fails

Now,I am struggling to formulate when that caching is really beneficial inside `CFile::Open` and when it brings more problems (like here)

**Comment by basilgello** [comment]
The issue with these "dynamic" HTTP/S addresses is that every dynamic video URL would be refused tomopen just because the "static" part of video URL is treated as a real fileststem path. Maybe advanced setting disabling directory cache would be the solution?

**Comment by basilgello** [comment]
This helps for now:

```patvh
diff --git a/xbmc/filesystem/File.cpp b/xbmc/filesystem/File.cpp                                                                                               
index 317816d..70cbf8a 100644                                                                                                                                  
--- a/xbmc/filesystem/File.cpp
+++ b/xbmc/filesystem/File.cpp
@@ -267,10 +267,12 @@ bool CFile::Open(const CURL& file, const unsigned int flags)
     if (url2.IsProtocol("apk") || url2.IsProtocol("zip") )
       url2.SetOptions("");

-    if (!g_directoryCache.FileExists(url2.Get(), bPathInCache) )
-    {
-      if (bPathInCache)
-        return false;
+    if (!URIUtils::IsInternetStream(file, false)) {
+      if (!g_directoryCache.FileExists(url2.Get(), bPathInCache) )
+      {
+        if (bPathInCache)
+          return false;
+      }
     }

     /*
```

**Comment by basilgello** [comment]
@arminfuerst Can you please revert the patch we did together and check if the PR above solves the issue?

**Comment by CrystalP** [comment]
>     2. Subtitle scanner attempts to scan http directory with no listing, sees 0 items inside and caches it once

I though I applied a fix to avoid scanning for external subtitles for upnp sources as it's not safe to manipulate the url in any way.

For a solution to this issue, please keep in mind actual http/s sources. The base url can be browsed, cached, scanned into the library (maybe same thing for webdav?).

**Comment by arminfuerst** [comment]
@basilgello After having slept one night, I have two thoughts on the fix:
1. Probably you should add a debug statment in case `bPathInCache` is true (around original Line 273 in File.cpp), sth. like "Directory 'bla' found in cache, but File 'xyz' is missing in cache.". If you would have seen this in the debug log in the first place, we would have been much faster. And if this statement causes other issues, debugging could be faster.
2. Probably the better solution would be to modify the cache when adding a directory. I assume when a directory is added to the cache, a file listing is done in this directory and all files are added to the cache as well. In case not a single file is found (or if possible: If directory listing returns an error / is not possible), simply don't add the directory to the cache. Having a directory in the cache without a single file in it imho has little advantage. And in case such a directory really exists, the performance impact should be little.

@CrystalP Point 2 could also address your concern about actual http/s sources?

**Comment by basilgello** [comment]
I have implemented idea no.2 in #25818 @arminfuerst ^^

**Comment by arminfuerst** [comment]
> I have implemented idea no.2 in #25818 @arminfuerst ^^

Thanks, my plan is to checkout this branch, compile it from there and test that version. But I can only do it in the evening.

**Comment by basilgello** [comment]
Perfect, then you tell me here or by email if that works or not, if it does I kick LE12 / RPi2 builds for you

**Comment by basilgello** [comment]
@anohren Please check your email, I prepared another build for you :)

**Comment by arminfuerst** [comment]
@basilgello I just verified, the current fix from PR #25818 is working on my machine. Since in the PR there is an ongoing discussion about a different solution, I can easily test them if you push them to this branch.

**Comment by basilgello** [comment]
Perfect!

**Comment by anohren** [comment]
Well, I feel both thick and blind because I don't understand how it'd help to reproduce it on Android, and I still couldn't spot the hack you two were referring to 😔 I couldn't see much more than a blame-link. Anyway, it probably doesn't matter:

An update: I do experience it as I described on macOS, i.e. immediate error message, **but** I took a closer look on Debian and it appears that the first video is playing for a fraction of a second (sometimes — it's a slow machine) before disappearing and I mistook that for not playing. The subsequent video results in the error message, just as you describe. I think that qualifies it. 

After this I took another look at OSMC (which I haven't really used UPnP with for a while since this issue has been going on for a year) and you're right, the first video plays there, so it's also as described in the OP. 

I've mainly been on macOS and have always reached for it first when testing, so that misled me a bit (I've always treated all devices as exhibiting the same defect). Sorry for the confusion. 

In summary: two of my devices follow the pattern in this issue, but my two macs don't appear to. I think it's possibly the same issue on macOS as well even if it doesn't show. I'm happy to just put macOS on ice until I've tried the existing fix on Debian, and if the fix ends up in a nightly build I can try it on macOS eventually. No hurry. I hope that'll minimize the amount of work in case it is in fact the same issue. 

(That the first video only plays for a fraction of a second on Debian also seems like something that can wait until after testing a fix)

**Comment by arminfuerst** [comment]
@anohren Probably I confused you with the Android question: I understood your reproduction as you initiate the UPnP request to KODI from an IOS device. As I don't have one, I asked whether you could initiate the request from an Android device, then I could try to reproduce it.
The first video is working without any issues on my devices, it plays also for hours, I can pause, rewind, ...

**Comment by anohren** [comment]
I can confirm that the problem is nonexistent on macOS with the oct 18 nightly of v21.

**Comment by basilgello** [comment]
Yes, there will be definitely more point releases!
